### PR TITLE
HTL-10186: fix qs vulnerability

### PR DESCRIPTION
**Pull request by trousdalew**
Update/remove packages to close qs vulnerability

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "An application framework based on Nodejs and Angularjs designed to bring together all of the ingredients needed to create structured, dynamic, large-scale web apps.",
   "main": "lib/server/wilson.js",
   "scripts": {
-    "test": "gulp test"
+    "test": "gulp test",
+    "build": "gulp build"
```
Comment (gaurav-pal1) [inline package.json:8]: where is npm build getting called?

Comment (trousdalew) [reply package.json:8]: I think build scripts must directly use gulp build, but it's helpful to have as a command for developers

Comment (trousdalew) [reply package.json:8]: I guess it will be obsolete soon

**Comment by gaurav-pal1** [comment]
👍

**Pull request merged**